### DISCUSSION TITLE
"Re-try connection" on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by jarrellmark** [iongion/container-desktop#10] [Q&A]
Hi,

I'm getting an error called "Re-try connection" after loading the program on Ubuntu 20.04. It never gets past that error.

I installed podman like this:

https://podman.io/getting-started/installation

```
. /etc/os-release
echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
curl -L "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key" | sudo apt-key add -
sudo apt-get update -qq
sudo apt-get -qq -y install podman
```

I installed podman-desktop-companion like this:

snap install --dangerous podman-desktop-companion_3.4.2-alpha.2_amd64.snap

Thanks!

**Comment by iongion** [comment]
Thanks @jarrellmark - I actually would use any help here, the alpha builds are not tested, except the appimage (which should only be used for testing/evaluation).

I don't run on Ubuntu, so I cannot tell you much.

There should be logs created by podman-desktop-companion in your home directory `~/.config/podman-desktop-companion/logs/*.log`

You can share, but before attaching them here, make sure they don't contain any secrets / sensitive information.

**Comment by jarrellmark** [comment]
Thanks @iongion ,

I don't have a folder called `~/.config/podman-desktop-companion/`.

There is a folder called `~/snap/podman-desktop-companion`, but it's kind of empty. The contents look like this:

```
~/snap/podman-desktop-companion$ ls -laR
.:
total 8
drwxr-xr-x  2 mark mark 4096 Jan  9 13:08 .
drwx------ 13 mark mark 4096 Jan  9 12:58 ..
lrwxrwxrwx  1 mark mark    2 Jan  9 12:58 current -> x1
```

current is a symlink pointing to x1, but x1 doesn't exist.

Thanks!
